learningobjects/tests/test_api.py

```diff
@@ -0,0 +1,62 @@
+"""
+Test api functions.
+"""
+
+from django.test.testcases import TestCase
+
+from django.contrib.auth.models import User
+
+from learningobjects.models import Course
+from learningobjects.api import create_course
+
+# pylint: disable=no-member
```

Comment:
These are in almost all of these, what is triggering these?  If it is super common we should just add it to the pylintrc, but it would be nice to have a description as to why, i.e. `# matplotlib dynamically adds members to its imports, so disabling`.  Also, these are covering the whole file, is there a way to just disable a line item?